force-app/main/default/classes/PMT_Payment_TDTM.cls

```diff
@@ -321,4 +323,39 @@ public class PMT_Payment_TDTM extends TDTM_Runnable {
             }
         }
     }
+
+    private void handleAllocationsAndOpportunities(
+        List<npe01__OppPayment__c> newList,
+        List<npe01__OppPayment__c> oldList
+    ) {
+        List<npe01__OppPayment__c> refunds = new List<npe01__OppPayment__c>();
+        List<npe01__OppPayment__c> oldRefunds = new List<npe01__OppPayment__c>();
+        for (Integer i = 0; i < newList.size(); i++) {
+            if (newList[i].npe01__Payment_Amount__c < 0 && String.isNotBlank(newList[i].Elevate_Payment_ID__c)) {
```

Comment:
Exclude Elevate payment because we don't want the opp to be updated twice if the Connector package version is outdated.